config/env/production.js

```diff
@@ -47,13 +54,14 @@ if (AWSCreds) {
 
 // If running in Cloud Foundry with a redis service
 if (redisCreds) {
-  module.exports.session = {
-    adapter: 'redis',
+  session = _.extend({}, session, {
+    adapter: 'socket.io-redis',
     host: redisCreds.hostname,
     port: redisCreds.port,
-    db: 0,
+    db: 1,
```

Comment:
No clue, it was set to '1' in the original file! I guess that is db it uses in production? Since its a magic number, it should at least be in a variable (and probably be configurable), but I didn't want to mess with it without being able to test it.